src/components/EditForm.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class TodoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    const todo = this.props.prevTodo;
```

Comment:
Since we're receiving props in the constructor, use that instead of `this.props....`
```js
const todo = props.prevTodo;
```